spec.md

```diff
@@ -173,6 +173,85 @@ minimally trusted storage servers is required.
 
 ## Authorization
 
+The ability to do cross-domain, operator-independent, standardized cloud storage
+operations requires an authorization system that is:
+
+* Modular and layered (for future agility / upgradability)
+* Not limited to traditional domain-based "usernames and passwords"
+* A hybrid, using object capability principles at its baseline, but also able
+  to provide ACL or RBAC-like functionality for user convenience. In other words,
+  the system needs to support both "anyone with the link can..." and "these are
+  specific people and groups allowed to..." styles of access control
+* Compatible with cross-domain replication
+* Compatible with end-to-end client side encryption (but also not rely on
+  encryption as the sole authorization method)
+* "Private by default".
+  That is, by default, unless otherwise specified, only the controller of a
+  space (or of a collection or resource) is authorized to perform any operation
+  (read, write, delete, etc)
+
+As the state of the art in cross-domain authorization advances, we expect there
+to be multiple profiles and specs that could be used to perform WAS API
+calls. However, to start with, this specification will focus on a single minimal
+authorization profile.
+
+### WAS Authorization Profile v0.1
+
+The initial W.A.S. Authorization Profile uses the following specifications.
+
+1. Identity (for controllers or clients/agents): [DID 1.1](https://www.w3.org/TR/did-1.1/)
+   or [CID 1.0](https://www.w3.org/TR/cid-1.0/) identifiers.
+2. Capability data model: [Authorization Capabilities for Linked Data v0.3](https://w3c-ccg.github.io/zcap-spec/)
+3. Authorization request protocol: Out of scope (implementers are encouraged to
+   use VC-API, OpenId4VP, or GNAP, as appropriate)
+4. Proof of Possession / authorization invocation: [HTTP Signatures (Cavage
+   draft 12)](https://datatracker.ietf.org/doc/html/draft-cavage-http-signatures)
```

Comment:
RFC 9421 ok?
https://www.rfc-editor.org/rfc/rfc9421.html

Reply:
Tragically, no! All the DB libraries we're using are coded against Draft 12, specifically. And between Draft 12 and the RFC there were numerous breaking changes. So one of our goals, in the WAS Authorization Profile v0.2 will be to bring the zCap spec (and libraries) up to speed to use the final RFC

Reply:
Hmm but why don't we just specify the RFC behavior, and note the lack of exact conformance in implementations that use the old libraries? IMHO it should be possible to conform to the normative language here by using the RFC9421 and not the draft 12. Spec can say MUST support the RFC9421 syntax and MAY support draft 12

Reply:
@gobengo ok, changed it to MUST - RFC 9421, MAY - cavage draft 12.